src/custom/pages/Faq/Faq.md

```diff
@@ -0,0 +1,117 @@
+
+### What token pairs does CowSwap allow to trade?
+Any valid ERC20 token pair for which there is some basic liquidity on a DEX (like Uniswap or Balancer).
+
+### Why is CowSwap able to offer gas-free trades?
+CowSwap is able to offer gas-free trades because the orders are submitted off-chain via signed messages. Once you approve your funds for spending on the dapp, you can submit orders via signed messages that contain the trade’s details, such as limit price, amount, timestamp, and so on. 
+
+### Do I need ETH to trade?
+For the trade itself you do not need to hold ETH. Although, in order to be able to trade on CowSwap, you first need to approve your funds for spending on the dapp<small>**</small>. For that action, you need ETH to pay for gas fees. Once you’ve done this, ETH is no longer required as CowSwap charges the fee from the sell token. 
+
+
+<small>** In the neartime future, if you are trying to sell an ERC20 that allows offline approvals, then the ETH needed to pay for allowing your funds to be spent is not needed anymore, making the trading experience fully gas-free. Keep in mind that this is only possible with ERC20 tokens that have such functionality; if not, you will need ETH to execute the approval transaction only.</small>
+
+### How does a trader submit a valid order in CowSwap?
+In order for a trader to submit a valid order to CowSwap, they must do the following steps:
+
+1. Approve the CowSwap smart contract to spend the token on your behalf. By executing this smart contract interaction you are approving the contract to withdraw the funds from your wallet once the trade you have signed has been filled in a batch auction. 
+2. Once the approval has been mined, the next step is to sign a meta-tx in which you will see the parameters of the order you are about to place in the CowSwap interface. After that, there's nothing else to do. 
+3. Once the order is executed, you will see a notification in the CowSwap UI and hear a confirming “Moo” sound.  
+
+### Why does the UI dapp have a warning ”Fees exceed From amount”?
+In order for solvers (order settlement solution providers) to be economically viable, they need to take into account how much gas they spend executing the settlement transaction. The protocol’s fee ensures that solvers are incentivized to include the order in a settlement (similar to how gas is paid on traditional DEXs). The fee is directly taken from the sell amount, which therefore has to have a certain minimum size.
+
+### Why do I need to approve a token before trading?
+When an order is executed, the settlement contract withdraws the sell amount from the trader’s token balance via the Allowance Manager (for more information cf. Smart Contract Architecture). In order to allow that to happen, the trader has to first approve the Allowance Manager contract to spend tokens on their behalf. The smart contract logic ensures that no token can be spent without deliberately signing an order for it.
```

Comment:
Shouldn't we have link on `Smart Contract Architecture`?

Reply:
Makes sense. Pointing to the smart contracts for now. Probably good to use the dev portal later on.